ext/pdo_oci/tests/bug60994.phpt

```diff
@@ -0,0 +1,118 @@
+--TEST--
+PDO OCI Bug #60994 (Reading a multibyte CLOB caps at 8192 characters)
+--SKIPIF--
+<?php
+/* $Id$ */
+if (!extension_loaded('mbstring') || !extension_loaded('pdo') || !extension_loaded('pdo_oci')) die('skip not loaded');
+require dirname(__FILE__).'/../../pdo/tests/pdo_test.inc';
+if (!strpos(strtolower(getenv('PDOTEST_DSN_UTF8')), 'charset=al32utf8')) die('skip expected output valid for AL32UTF8 character set');
```

Comment:
We are not using a different DSN for UTF8. DSN string will be changed manually to run UTF8 tests. Hence, please change "PDOTEST_DSN_UTF8" to "PDOTEST_DSN".